src/routes/safe/components/Apps/api/config-service.ts

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios'
-
+import { getNetworkId } from 'src/config'
 import { SAFE_APPS_LIST_URL } from 'src/utils/constants'
```

Comment:
Sorry, one more thing about this service.
There will be new endpoints in future and would make sense to treat this module as an SDK for the config service.
So I'd suggest
* instead of `SAFE_APPS_LIST_URL` to have a constant `CONFIG_SERVICE_URL` (which will be `https://safe-config.staging.gnosisdev.com/api/v1`)
* each endpoint to be defined as a constant in this file (`config-service.ts`). Like so:
```
enum Endoints = {
  SAFE_APPS = '/safe-apps/`
}
```

Reply:
Great, i'll do that